contrib/metadata-ingestion/README.md

```diff
@@ -13,9 +13,9 @@ i split the ingestion procedure to two part: [datahub-producer] and different [m
 - [X] datahub-producer load json avro data.
 - [X] add lineage-hive generator
 - [X] add dataset-jdbc generator[include [mysql, mssql, postgresql, oracle] driver]
-- [ ] enhance dataset-jdbc generator [hive-driver]
+- [X] add dataset-hive generator
+- [*] add lineage-oracle generator
```

Comment:
I assume you wanted `[ ]` here since you haven't started working on this item yet?

Reply:
i already do some oracle part work, but it will be treat as external dependency project.
i will plan to finish in one month.

https://github.com/clojurians-org/simple-datahub/blob/master/contrib/metadata-etl/library/extension/Oracle.hs

Reply:
i will change to [*] to [ ]

Reply:
you mean those implemeneted file?  i switch to  python version for it
change
[ ] enhance dataset-jdbc generator [hive-driver]
to
[X] add dataset-hive generator

https://github.com/clojurians-org/datahub/blob/hive-generator/contrib/metadata-ingestion/bin/dataset-hive-generator.py

https://github.com/clojurians-org/datahub/blob/hive-generator/contrib/metadata-ingestion/bin/dataset-hive-generator.py.nix

Reply:
sorry, i see what you said.
the origin file https://github.com/linkedin/datahub/blob/master/metadata-ingestion/hive-etl/hive_etl.py i migrate didn't implement the columns information.
i will add it.

Reply:
i only migrate
from 
https://github.com/clojurians-org/datahub/tree/hive-generator/metadata-ingestion/hive-etl
to
https://github.com/clojurians-org/datahub/blob/hive-generator/contrib/metadata-ingestion/bin/dataset-hive-generator.py
https://github.com/clojurians-org/datahub/blob/hive-generator/contrib/metadata-ingestion/bin/dataset-hive-generator.py.nix

is there extra things i need to handle?